content/blog/2022/03/2022-03-25-systemd-migration.adoc

```diff
@@ -0,0 +1,156 @@
+and also run jobs directly on the controller,
+a practice which the Jenkins project link:https://www.jenkins.io/doc/book/security/controller-isolation/[explicitly discourages].
+When in doubt, secure the drop-in unit by settings its permissions to 0600 with `chmod(1)`.
+
+One benefit of the drop-in unit is that it unifies configuration across all three distribution: Debian, Red Hat, and openSUSE.
+Gone are the days of maintaining distribution-specific configuration logic.
+
+Also note that the drop-in unit is not overwritten on upgrades.
+Gone are the days of getting conflicts in `/etc/{default,sysconfig}/jenkins` on upgrades,
+at least _after_ the upgrade to a `systemd(1)`-based package is completed.
+
+NOTE: Unlike the System V `init(8)` configuration, the `override.conf` file only contains customizations, not the original defaults.
+Users who are accustomed to editing an existing set of defaults must refer to the (read-only) service unit side-by-side when editing the drop-in unit
+or use a command like `systemctl edit jenkins --full`, which copies the original service unit instead of creating a drop-in unit.
+
+A final point to mention about the service unit is its use of specifiers,
+which may be unfamiliar to some users.
+The `systemd(1)` documentation contains link:https://www.freedesktop.org/software/systemd/man/systemd.unit.html#id-1.13.3[a table of specifiers available in unit files].
+
+==== Migration
+
+The Jenkins project ships logic to automatically migrate the System V `init(8)` configuration file to the new `systemd(1)` `override.conf` format.
+This migration logic does nothing if an `override.conf` file already exists,
+which would be an indication that the migration script already ran
+or that the user has made their own customizations that should be preserved.
+If `override.conf` does not exist, package installation migrates the old System V `init(8)` configuration file to `override.conf`.
+
+==== Logging
+
+The `systemd(1)` package also uses `systemd-journald(8)` for logging by default.
+Rather than creating a log file in `/var/log/jenkins.log`,
+Jenkins now logs to the system's journal.
+Log entries may be viewed with `journalctl -u jenkins`.
+This is perhaps the most noticeable user-visible change in this migration.
+
+==== Startup notifications
+
+The System V `init(8)` logic was asynchronous; i.e., running `/etc/init.d/jenkins start` would return prior to the completion of Jenkins startup.
+The `systemd(1)` logic is synchronous; i.e., running `systemctl start jenkins` will block until Jenkins signals successful startup.
+This allows system administrators to write automation to programmatically deploy Jenkins using modern tools like Ansible.
+
+=== Reporting issues
+
+If you find a regression, please file a bug report in link:https://issues.jenkins.io/[Jira].
+When reporting an issue, include the following information:
+
+. Use the `core` component.
```

Comment:
I've seen a number of other packaging related issues reported in the packaging project. Is there a specific reason you chose core rather than packaging? (non-blocking question, I'm asking for my information, not to block the pull request)

```suggestion
. Use the `packaging` component.
```